stacker/lookups/handlers/output.py

```diff
@@ -27,11 +27,11 @@ def handler(value, provider=None, context=None, fqn=False, rfqn=False,
 
     """
 
-    if rfqn and context.environment.get('namespace'):
+    if rfqn and context.environment['namespace']:
```

Comment:
Sorry, I realize I might have been a little confusing in my original comment.  You should be able to change this line to:

```
if rfqn:

```

Since we always ensure that we have a namespace, it's required in the context.